src/emissions/emissions.service.ts

```diff
@@ -89,36 +88,34 @@ export class EmissionsService {
     const date = new Date(periodDate);
     const month = date.getUTCMonth() + 1;
 
-    const isValidDevQuery = () => {
-      return (
+    if (queryResult === undefined) {
```

Comment:
Reverting this back to the mess it was before.

Original changes: https://github.com/US-EPA-CAMD/easey-emissions-api/commit/a617780710624fa83f1d688765016e0ad6da69c7